docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc

```diff
@@ -1,12 +1,16 @@
 [id="troubleshooting-expired-certificates_{context}"]
 = Troubleshooting expired certificates
 
-Each released `{bin}` binary includes an embedded system bundle that expires 30 days after the release due to certificates embedded in the OpenShift cluster.
+Prior to {rh-prod} 1.2.0, each released `{bin}` binary included a system bundle that expires 30 days after the release due to certificates embedded in the OpenShift cluster.
 Using an older `{bin}` binary or system bundle will result in an expired certificates error.
 
+Starting from {prod} 1.2.0, the embedded certificates can be automatically renewed by {bin}.
+The certificate renewal process is triggered when needed when [command]`{bin} start` is run.
+When certificates need to be renewed, this can adds up to 5 minutes to the cluster startup time.
```

Comment:
We try to avoid phrasal verbs (for example, "startup"). Reword to "this can add up to five minutes to the start time of the cluster"?